src/utils/validator.js

```diff
@@ -0,0 +1,25 @@
+import { ERROR_MESSAGE } from '../constants/systemMessages.js';
+
+const validator = Object.freeze({
+  carNames(carNames) {
+    if (carNames.length !== new Set(carNames).size)
+      throw new Error(ERROR_MESSAGE.DUPLICATE_NAME);
+    if (carNames.some((name) => name.length > 5))
+      throw new Error(ERROR_MESSAGE.INVALID_NAME_LENGTH);
+    if (carNames.some((name) => name.includes(' ')))
+      throw new Error(ERROR_MESSAGE.INVALID_NAME_SPACE);
+    if (carNames.some((name) => !/^[a-zA-Z]+$/.test(name)))
+      throw new Error(ERROR_MESSAGE.INVALID_NAME_CHARACTER);
+    if (carNames.length < 2) throw new Error(ERROR_MESSAGE.INVALID_CAR_COUNT);
+  },
+
+  count(count) {
+    if (count.includes(' ')) throw new Error(ERROR_MESSAGE.INVALID_COUNT_SPACE);
+    if (!/^[0-9]+$/.test(count))
+      throw new Error(ERROR_MESSAGE.INVALID_COUNT_CHARACTER);
+    if (Number(count) < 1) throw new Error(ERROR_MESSAGE.INVALID_COUNT_MIN);
+    if (Number(count) > 1000) throw new Error(ERROR_MESSAGE.INVALID_COUNT_MAX);
```

Comment:
```suggestion
    if (Number(count) > 1_000) throw new Error(ERROR_MESSAGE.INVALID_COUNT_MAX);
```

네 자리가 넘어가는 숫자들은 numeric sperator를 이용해보시면 좋을거 같아요 ㅎㅎ

Reply:
numeric separator라는 게 있었군요..! 적용해보겠습니다 ㅎㅎ